lte/gateway/python/magma/pipelined/policy_converters.py

```diff
@@ -193,3 +210,17 @@ def _get_direction_for_match(flow_match):
 def convert_ipv4_str_to_ip_proto(ipv4_str):
     return IPAddress(version=IPAddress.IPV4,
                      address=ipv4_str.encode('utf-8'))
+
+
+def convert_ipv6_bytes_to_ip_proto(ipv6_bytes):
+    return IPAddress(version=IPAddress.IPV6,
+                     address=ipv6_bytes)
+
```

Comment:
Can we add typing for both to differentiate ipv4 and ipv6 param types

Reply:
I'll do a clean up PRa bit later, there are a few places that need annotations